vng_api_common/audittrails/viewsets.py

```diff
@@ -2,6 +2,7 @@
 
 from django.db import transaction
 from django.http import Http404
+from drf_spectacular.utils import extend_schema, OpenApiParameter
```

Comment:
Deze wijzigingen in dit bestand voegen op dit moment weinig toe, deze kun je weghalen